tests/integration.rs

```diff
@@ -4356,6 +4356,97 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        fn get_balloon_mem(api_socket: &str) -> u64 {
+            let (cmd_success, cmd_output) = remote_command_w_output(&api_socket, "info", None);
+            assert!(cmd_success);
+
+            let info: serde_json::Value = serde_json::from_slice(&cmd_output).unwrap_or_default();
+            let total_mem = &info["config"]["memory"]["size"]
+                .to_string()
+                .parse::<u64>()
+                .unwrap();
+            let actual_mem = &info["memory_actual_size"]
+                .to_string()
+                .parse::<u64>()
+                .unwrap();
+            total_mem - actual_mem
+        }
+
+        #[test]
+        #[cfg(target_arch = "x86_64")]
+        fn test_virtio_balloon() {
+            let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+            let guest = Guest::new(Box::new(focal));
+            let mut workload_path = dirs::home_dir().unwrap();
+            workload_path.push("workloads");
```

Comment:
> These two lines can be removed since `workload_path` is never used.

ok

> @ShirleyFei the PR looks quite good. I have several suggestions to improve it, but after you addressed them, we should be good to go and merge the PR :)

Thanks very much for the detail review.  I will polish all the comments (some of them are not replied, but all are admitted :) ), and push a new version later.

Reply:
> These two lines can be removed since `workload_path` is never used.

Notice there are still many other unused workload_path() in tests/integration.rs, I will submit a new PR to clean up them later. :)

Reply:
>Notice there are still many other unused workload_path() in tests/integration.rs, I will submit a new PR to clean up them later. :)

That'd be awesome :)